components/org.wso2.carbon.identity.application.authenticator.biometric.servlet/pom.xml

```diff
@@ -0,0 +1,147 @@
+
+    <parent>
+        <groupId>org.wso2.carbon.identity.outbound.auth.biometric</groupId>
+        <artifactId>identity-outbound-auth-biometric</artifactId>
+        <relativePath>../../pom.xml</relativePath>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>org.wso2.carbon.identity.application.authenticator.biometric.servlet</artifactId>
+    <packaging>bundle</packaging>
+    <name>WSO2 Carbon - Endpoint for Biometric Authenticator</name>
+    <description>A custom wso2 products or solution</description>
+    <url>http://www.wso2.com</url>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.service.component.annotations</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.service.http</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.service.component</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.equinox</groupId>
+            <artifactId>org.eclipse.equinox.http.helper</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>org.wso2.carbon.identity.application.authenticator.biometric.servlet.internal</Private-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.identity.application.authenticator.biometric.servlet.internal,
+                            org.wso2.carbon.identity.application.authenticator.biometric.servlet.*
+                        </Export-Package>
+                        <Import-Package>
+                            javax.servlet; version="[2.6.0,3.6.0)",
+                        </Import-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
We need to remove the dynamic importing if possible and define each of the import packages separately with tested versions, under the `<Import-Package>` tag. One key benefit of this would be that it is easier to troubleshoot if any dependency issue arises with a future release.